### PR TITLE
Back to httpstub mocked unit tests

### PR DESCRIPTION
**Pull request by corinnekrych**
This PR bring back unit tests using aerogear-ios-httpstub to mock http call
@matzew do you mind to have a look?

### CODE REVIEW

**Comment by matzew** [comment]
I am on Xcode: Version 6.1 (6A1046a) and I am getting this:

```
/Users/matzew/TEMP/iOS/aerogear-ios-http-1/AeroGearHttp/MultiPartData.swift
/Users/matzew/TEMP/iOS/aerogear-ios-http-1/AeroGearHttp/MultiPartData.swift:43:42: Operand of postfix '!' should have optional type; type is 'String'
/Users/matzew/TEMP/iOS/aerogear-ios-http-1/AeroGearHttp/MultiPartData.swift:44:46: Operand of postfix '!' should have optional type; type is 'String'
```

**Comment by corinnekrych** [comment]
@matzew you should compile with xcode6.1.1(or xcode 6.2 beta) the stable release as stated in README

**Comment by matzew** [comment]
oh, did not see the note :)

installing 6.1.1

On Fri, Jan 16, 2015 at 10:19 AM, Corinne Krych <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> you should compile with xcode6.1.1(or
> xcode 6.2 beta) the stable release as stated in README
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-ios-http/pull/33#issuecomment-70227318>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by matzew** [comment]
w/ 6.1.1 works on my machine(tm)

:+1:

File: AeroGearHttpTests/HttpTests.swift [hunk 1]
```diff
@@ -15,195 +15,166 @@
...
 class HttpTests: XCTestCase {
 
+    func httpStubResponseWithInputParams(request: NSURLRequest!, status: Int, params:[String: AnyObject]?) -> StubResponse {
+        var data: NSData
+        if ((params) != nil) {
+            data = NSJSONSerialization.dataWithJSONObject(params!, options: nil, error: nil)!
+        } else {
+            data = NSData()
+        }
+        return StubResponse(data:data, statusCode: status, headers: ["Content-Type" : "text/json"])
```
Comment (matzew) [inline AeroGearHttpTests/HttpTests.swift]: should it be `application/json` ?

**Comment by matzew** [comment]
great seeing the unit tests come back

**Pull request merged**